packages/core/config/storybook/preview.js

```diff
@@ -1,6 +1,5 @@
 const React = require('react')
-
```

Comment:
Called `webpack-merge`, but actually a more general package for deep-merging config structures. We already use it in this way in `storybook/main.js`.